docs/api/quest_manager.md

```diff
@@ -28,6 +28,8 @@ It handles all the [quest pools] and also provide helpers to add more pools or m
 | [**is_quest_completed**](#bool-is_quest_completedquest-quest)**(quest:** [Quest](/api/quest_resource.md)**)** | [bool](https://docs.godotengine.org/en/stable/classes/class_bool.html) |
 | [**is_quest_in_pool**](#bool-is_quest_in_poolquest-quest-pool_name-string)**(quest:** [Quest](/api/quest_resource.md), **pool_name:** [String](https://docs.godotengine.org/en/stable/classes/class_string.html)**)** | [bool](https://docs.godotengine.org/en/stable/classes/class_bool.html) |
 | [**call_quest_method**](#void-call_quest_methodquest_id-int-method-string--args-array)**(quest_id:** [int](https://docs.godotengine.org/en/stable/classes/class_int.html), **method:** [String](https://docs.godotengine.org/en/stable/classes/class_string.html),  **args:** [Array](https://docs.godotengine.org/en/stable/classes/class_array.html)**)** | **void** |
+| [**set_quest_property**](#void-set_quest_propertyquest_id-int-property-string--value-variant)**(quest_id:** [int](https://docs.godotengine.org/en/stable/classes/class_int.html), **property:** [String](https://docs.godotengine.org/en/stable/classes/class_string.html),  **value:** [Variant](https://docs.godotengine.org/en/stable/classes/class_variant.html)**)** | **void** |
+| [**get_quest_property**](#void-get_quest_propertyquest_id-int-property-string)**(quest_id:** [int](https://docs.godotengine.org/en/stable/classes/class_int.html), **property:** [String](https://docs.godotengine.org/en/stable/classes/class_string.html)**)** | **void** |
```

Comment:
The return type of the get_quest_property here is wrong. It should be `Variant` and not void.